.gitignore

```diff
@@ -153,3 +153,6 @@ checklink/cookies.txt
 *.csv
 *.parquet
 nbs/_extensions
+
+# VSCode
+*.code-workspace
```

Comment:
this could actually be useful if setup correctly
```suggestion
```

Reply:
Wont people have their individual preferences in this file?